guides/source/active_job_basics.md

```diff
@@ -131,77 +119,274 @@ That's it!
+end
 ```
 
-[`config.active_job.queue_adapter`]: configuring.html#config-active-job-queue-adapter
+This ensures that only one job for a given contact can run at a time, regardless of the job class.
 
-### Starting the Backend
+Read more about [Concurrency Controls in the Solid Queue documentation](https://github.com/rails/solid_queue?tab=readme-ov-file#concurrency-controls).
 
-Since jobs run in parallel to your Rails application, most queuing libraries
-require that you start a library-specific queuing service (in addition to
-starting your Rails app) for the job processing to work. Refer to library
-documentation for instructions on starting your queue backend.
+### Error Reporting on Jobs
 
-Here is a noncomprehensive list of documentation:
+If your error tracking service doesn’t automatically report job errors, you can manually hook into Active Job to report exceptions. For example, you can add a `rescue_from` block in `ApplicationJob`:
 
-- [Sidekiq](https://github.com/mperham/sidekiq/wiki/Active-Job)
-- [Resque](https://github.com/resque/resque/wiki/ActiveJob)
-- [Sneakers](https://github.com/jondot/sneakers/wiki/How-To:-Rails-Background-Jobs-with-ActiveJob)
-- [Queue Classic](https://github.com/QueueClassic/queue_classic#active-job)
-- [Delayed Job](https://github.com/collectiveidea/delayed_job#active-job)
-- [Que](https://github.com/que-rb/que#additional-rails-specific-setup)
-- [Good Job](https://github.com/bensheldon/good_job#readme)
-- [Solid Queue](https://github.com/rails/solid_queue?tab=readme-ov-file#solid-queue)
+```ruby
+class ApplicationJob < ActiveJob::Base
+  rescue_from(Exception) do |exception|
+    Rails.error.report(exception)
+    raise exception
+  end
+end
+```
+
+If you use ActionMailer, you’ll need to handle errors for `MailDeliveryJob` separately:
+
+```ruby
+class ApplicationMailer < ActionMailer::Base
+  ActionMailer::MailDeliveryJob.rescue_from(Exception) do |exception|
+    Rails.error.report(exception)
+    raise exception
+  end
+end
+```
+
+### Transactional Integrity on Jobs
+
+By default, Solid Queue uses a separate database from your main application. This avoids issues with transactional integrity, which ensures that jobs are only enqueued if the transaction commits.
+
+However, if you use Solid Queue in the same database as your app, you can enable transactional integrity with Active Job’s `enqueue_after_transaction_commit` option which can be enabled for individual jobs or all jobs through `ApplicationJob`:
+
+```ruby
+class ApplicationJob < ActiveJob::Base
+  self.enqueue_after_transaction_commit = true
+end
+```
+
+You can also configure Solid Queue to use the same database as your app while avoiding transactional integrity issues by setting up a separate database connection for Solid Queue jobs.Read more about [Transactional Integrity in the Solid Queue documentation](https://github.com/rails/solid_queue?tab=readme-ov-file#jobs-and-transactional-integrity)
+
+### Recurring Tasks
```

Comment:
All of these subsections starting at line 281, I have the urge to cut down to one paragraph or one sentence even or remove entirely. (They are all well written! nothing to do with that and making them to the README and linking to the README is a good approach!) But I have take on the perspective of a dev who came here to quickly learn what's relevant while they're implementing Solid Queue for the first time, I would want less words :) Though I don't know enough yet...to come with those shorter words (normally I'd want to spin up a sample project and implement it and all that). 

So, given the timeframe, we can leave it as is for now and revisit next year. It could be case of "I didn't have time to write a shorter letter, so I wrote a long one instead" :)